src/app/Main/Body/QueryPage/Results/Result-MainHeader.component.tsx

```diff
@@ -29,13 +25,17 @@ const tableTitleStyle = (
     display: ${isTitle ? "flex" : "none"};
     width: ${isRow ? "inherit" : `${width}px`};
     height: ${isRow ? `${height}px` : "inherit"};
-    margin: 0;
-    padding: ${isRow ? "0 0.3em" : "0.2em 0"};
+    min-height: max-content;

```

Comment:
Consider evading working draft css attributes. They can cause compatibility issues. See:
https://developer.mozilla.org/en-US/docs/Web/CSS/min-height
In general check javascript/html/css features for compatibility before using them. MDN has a nice compatibility table for each feature.

Reply:
According to the table, "min-height: max-content" only causes problems when using internet explorer, so I see no reason not to use it?

Reply:
You right, but it is still a working draft. The behavior can change anytime and browsers can introduce bugs because of changes under the hood. I've had problems with the box sizing keywords myself and removed them. But you can use them if you are aware of the problems that might occur.